.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
is this a valid json comment?

Reply:
json doesn't have comment, but it's a launch file in vscode and it accepts that comment.
In fact, the whole file is generated by vscode when I create new launch file, then I update / add some more configs

Reply:
got it, thanks!